README.md

```diff
@@ -1,42 +1,80 @@
-  - Development of the idea will continue on the repository until it's ready to be integrated into another incubation or promoted out of incubation state (e.g., to a WG).
-  
-## What belongs here vs. in another repo/standards group/issue tracker
-There are a lot of different places that ideas for web standards could be proposed, so at times it may not be clear where the best place is to start. There are no hard and fast rules, but here are some things to consider:
+## My proposal Issue has widespread support and we've found consensus. Now what?
 
-  - If your proposal isn't in some way related to Immersive Web technologies (Virtual Reality, Augmented Reality, Mixed Reality, etc.), then this probably isn't the place for it. Consider checking out the [W3C Community and Business Groups](https://www.w3.org/community/) if you don't know of an existing incubation or specification that your proposal fits under. The [Web Incubator Community Group (WICG)](https://wicg.io/) is a good general-purpose venue.
-  - Similarly, if your proposal is related to the Immersive Web but useful in other contexts as well (i.e., Geolocation improvements), then it should be probably be handled by a group that's less exclusively focused on Immersive Web tech. Feel free to add a note where appropriate to let the Immersive Web community know about a relevant effort.
-  - If you have an issue with a specific spec, open an issue on that spec first.
+Terrific! You did the hard work of getting enough momentum in the group to figure out what you're making and how it would work. The next step is to recruit one or two people from other organizations who want to join you as feature advocates. This isn't a formal W3C community group role; it simply means that each of you agrees to do the work of nailing down a feature's details and working with the rest of the group to find consensus on its details.
 
-So what happens if you get it wrong? Nothing bad! We'll point you towards a more appropriate venue if necessary and, when possible, help migrate the issue for you. And please don't assume that a request to continue discussion of your issue elsewhere is a criticism, we genuinely want to ensure that your idea is reaching the right audience!
+Once two or three people from separate organizations have volunteered to be feature advocates then you should reach out to the CG chair and ask that a feature repo be created.
 
-## Code of Conduct
-**TL;DR: Treat other participants with respect or get banned.**
+## The CG chair made a new feature repo. Now what?
+
+One goal of the repo is to work through the nitty-gritty details of a feature with enough detail that it's clearly possible to implement and it's attractive to the relevant Working Group.
```

Comment:
"nitty-gritty" might be less understandable to a non-English-as-a-first-language participant.  "Specific details"?